hack/prepare-operatorhub-pr.sh

```diff
@@ -1,19 +1,19 @@
-#!/usr/bin/env -S bash -e
+#!/bin/bash -e
```

Comment:
why is this change necessary?

Reply:
The CI didnt have -S and the command was failing ..even after excluding S flag it was failing on CI

Reply:
Can you paste here the error? If needed we should update CI configuration - `using /usr/bin/env` has it advantages over `/bin/bash` drectly, because on some systems `bash` can be installed on a different location.

Reply:
I tried running it with the suggested usage nd the script was not able to execute, It threw the following error 
`Error: The action has timed out`

Reply:
that is not coming from the script, it is coming from GitHub actions - what was the run, can you share the link?

Reply:
https://github.com/redhat-developer/service-binding-operator/runs/1908927102?check_suite_focus=true

Reply:
Why I felt this commit triggered the timeout err is because only on this specific commit change I got the timeout issue, never before and not in the later commits as well.

Reply:
The script was stuck somewhere, hence the timeout. It would be good to add maybe some debugging infos to that script, so that you can see where it stucks. Please make shebang line `#!/usr/bin/env -S bash -e` and try again.

Reply:
Had experienced this earlier, getting the err again `/usr/bin/env: invalid option -- 'S'`